src/modules/config-handler/ConfigValidation.schema.ts

```diff
@@ -0,0 +1,10 @@
+import * as Joi from 'joi';
+
+export const configValidationSchema = Joi.object({
+  PORT: Joi.number()
+    .default(3000),
+  CDN_HOST: Joi.string()
+    .required()
+}).options({
+  allowUnknown: true
+});
```

Comment:
Just wondering if there is a better way to format this, feel free to ignore both of my suggestions if you prefer the current format.
```suggestion
export const configValidationSchema = Joi.object({
  PORT: Joi.number().default(3000),
  CDN_HOST: Joi.string().required()
}).options({
  allowUnknown: true
});
```

```suggestion
export const configValidationSchema = Joi.object({
  PORT: Joi
    .number()
    .default(3000),
  CDN_HOST: Joi
    .string()
    .required()
}).options({
  allowUnknown: true
});
```